apps/web-portal-client/lib/components/layout/nav-bar/index.jsx

```diff
@@ -1 +1,4 @@
 export * from './Navbar'
+export * from './Button'
```

Comment:
move `Button`, `Image`, `ImageShapes` directly under `lib/components` folder, because they are not specific for nav-bar